services/user.py

```diff
@@ -0,0 +1,68 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str | None = None,
+        first_name: str | None = None,
+        last_name: str | None = None
+) -> None:
+    if get_user_model().objects.filter(username=username).exists():
+        print("Username already exists")
+        return
+
+    users_data = {}
+    if email:
+        users_data["email"] = email
+    if first_name:
+        users_data["first_name"] = first_name
+    if last_name:
+        users_data["last_name"] = last_name
+
+    get_user_model().objects.create_user(
+        username=username,
+        password=password,
+        **users_data
+    )
+
+
+def get_user(user_id: int) -> User:
+    return get_user_model().objects.filter(pk=user_id).first()
+
+
+def update_user(
+    user_id: int,
+    username: str | None = None,
+    password: str | None = None,
+    email: str | None = None,
+    first_name: str | None = None,
+    last_name: str | None = None
+) -> None:
+    if not get_user_model().objects.filter(pk=user_id).exists():
```

Comment:
no need for this - if user not exists let the exception happen, and later when you work with django you may use get_object_or_404() method for cases like this. For now lets keep it simple